django/db/backends/mysql/schema.py

```diff
@@ -69,13 +69,29 @@ def _is_limited_data_type(self, field):
             and db_type.lower() in self.connection._limited_data_types
         )
 
+    def _is_text_or_blob(self, field):
+        db_type = field.db_type(self.connection)
+        return db_type is not None and (
+            db_type.lower().endswith("text") or db_type.lower().endswith("blob")
+        )
```

Comment:
`endswith()` accepts tuple
```suggestion
        return db_type and db_type.lower().endswith(("blob", "text"))
```

Reply:
thx